app/code/Magento/Catalog/Model/ImageUploader.php

```diff
@@ -188,19 +188,20 @@ public function getFilePath($path, $imageName)
     /**
      * Checking file for moving and move it
      *
-     * @param string $imageName
+     * @param string $tmpImageName
+     * @param string $baseImageName
      *
      * @return string
      *
      * @throws \Magento\Framework\Exception\LocalizedException
      */
-    public function moveFileFromTmp($imageName)
+    public function moveFileFromTmp($tmpImageName, $baseImageName)
```

Comment:
Unfortunately, we are not allowed to modify public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)